### ISSUE TITLE
`__init__.py` files missing?

### ISSUE THREAD

**Issue by dmitriy-serdyuk** [espnet/espnet#1831]
`__init__.py` files seem to be missing in all directories under `espnet/`. This causes various import errrors.

```
$ python -c "import espnet.utils"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'espnet.utils'
```

```
apply-cmvn --norm-vars=true data/train_nodev/cmvn.ark scp:exp/dump_feats/train/feats.1.scp ark:- 
Traceback (most recent call last):
  File "/network/home/serdyuk/projects/espnet/egs/timit/asr1/../../../utils/copy-feats.py", line 6, in <module>
    from espnet.transform.transformation import Transformation
ModuleNotFoundError: No module named 'espnet.transform'
```

**Comment by sw005320** [comment]
We did not assume such cases, but maybe this is a good idea to do it.
@kamo-naoyuki and @kan-bayashi, what do you think?

**Comment by kan-bayashi** [comment]
If we install without `--editable` option, I think this error will be happened.
https://github.com/espnet/espnet/blob/f70d6b163bce0d8743877faa654a8345dfdd56b2/setup.py#L113
In my understanding, `find_package` could not find them without `__init__.py`.
(We added at first but we removed them... I could not remember the reason...)

It is better to add `__init__.py` to all of the directories as the same as `espnet2/`.

**Comment by sw005320** [comment]
> It is better to add `__init__.py` to all of the directories as the same as `espnet2/`.

I think so.
Could you just add them?
I believe it would not have issues.

**Comment by kamo-naoyuki** [comment]
I didn't know we don't have it.  This should be a bug.  #1016 removed it.

**Comment by kamo-naoyuki** [comment]
TL;DR
If you don't indent to use `Namespace packages`, we must put __init__.py. 

(I think almost all people don't have to understand about __init__.py strictly.)

From python3.3, there are two types of module: https://docs.python.org/3/reference/import.html#regular-packages

- Regular packages: With __init__.py
- Namespace packages: Without __init__.py

To install `Namespace packages` from setup.py:  https://packaging.python.org/guides/packaging-namespace-packages/

**Comment by dmitriy-serdyuk** [comment]
> If we install without --editable option

Right, I installed without editable assuming that editable is only for development.

**Comment by ShigekiKarita** [comment]
Thanks for reporting this issue. #1842 fixed it.
